tests/algorithms/test_clock_angle.py

```diff
@@ -1,13 +1,21 @@
 import unittest
 
-from algorithm_services.algorithms.clock_angle import clock_angle
+from algorithm_services.algorithms.algorithm_factory import AlgorithmFactory
 
 
 class ClockAngleTestCase(unittest.TestCase):
+    def setUp(self):
+        self.teste_instance_0 = AlgorithmFactory.create_algorithm('clock_angle', [12, 0]) # noqa
```

Comment:
You testing the factory over and over, that should not be the point of this test.

Reply:
The purpose of this test case is to test the algorithm and not its factory; use ClockAngle class directly and create a new test case for AlgorithmFactory 💃